### PR TITLE
Send UPnP ports to the monitor

### LINKED ISSUE #1978: Upnp ports should be send to the monitor
When testing Upnp on golem windows 0.11.0 binaries I get the following warning:

```
2018-01-26 13:26:28 INFO     golem.network.upnp                  b'IGD: starting discovery'
2018-01-26 13:26:29 INFO     golem.network.upnp                  b'IGD: discovery complete: http://X.X.X.X:5000/ctl/IPConn'
2018-01-26 13:26:29 INFO     golem.network.upnp                  b'IGD: network configuration: {'local_ip_address': 'X.X.X.X', 'external_ip_address': 'X.X.X.X', 'connection_type': 'IP_Routed', 'status_info': ('Connected', 1601572, 'ERROR_NONE')}'
2018-01-26 13:26:29 INFO     golem.network.upnp                  b'IGD: mapped 40102 -> 40105 (TCP)'
2018-01-26 13:26:30 INFO     golem.network.upnp                  b'IGD: mapped 40103 -> 40106 (TCP)'
2018-01-26 13:26:30 INFO     golem.network.upnp                  b'IGD: mapped 3282 -> 3283 (TCP)'
2018-01-26 13:26:30 WARNING  golem.monitor                       b'Port status: 40102: unreachable, 40103: unreachable, 3282: unreachable'
```

This is caused by my mac node having the default ports forwarded.

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**
Resolves #1978

### CODE REVIEW

File: golem/client.py [hunk 1]
```diff
@@ -336,6 +336,8 @@ def start_network(self):
             self.node.pub_addr)
         hyperdrive_ports = self.daemon_manager.ports()
 
+        self.node.hyperdrive_prv_port = next(iter(hyperdrive_ports))
```
Comment (maaktweluit) [inline golem/client.py:339]: what is the added value of the `iter`? 
looks like this also can be `hyperdrive_ports[0]`

Comment (maaktweluit) [inline golem/client.py:339]: should we check if we received ports or will the manager handle this error?

**Review by maaktweluit — APPROVED** [review]
LGTM!

2 small q's, will still approve.

File: golem/client.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply golem/client.py:339]: `hyperdrive_ports` (currently one) are a set, so there is no indexing, and `next` requires an iterator.

Comment (mfranciszkiewicz) [reply golem/client.py:339]: A successful response from hyperdrive will contain its listening addresses / ports, unless it failed to start (and `requests` will throw a connection error). I could put a guard here but I believe it is not required.

Comment (maaktweluit) [reply golem/client.py:339]: Thank you, clear :)

Comment (maaktweluit) [reply golem/client.py:339]: Thank you, i agree :)

**Review by jiivan — APPROVED** [review]
lgtm

**Pull request merged**